ckanext/cfpb_extrafields/templates/package/resource_read.html

```diff
@@ -100,87 +117,141 @@ <h1 class="page-heading">{{ h.resource_display_name(res) | truncate(50) }}</h1>
+                        <option value='{"name":"plain_text", "ext":"txt"}'>text</option>
+                        <option value='{"name":"markdown", "ext":"md"}'>markdown</option>
+                        <option value='{"name":"css", "ext":"css"}'>css</option>
+                        <option value='{"name":"html", "ext":"html"}'>html</option>
+                        <option value='{"name":"java", "ext":"java"}'>java</option>
+                        <option value='{"name":"javascript", "ext":"js"}'>javascript</option>
+                        <option value='{"name":"matlab", "ext":"ml"}'>matlab</option>
+                        <option value='{"name":"perl", "ext":"perl"}'>perl</option>
+                        <option value='{"name":"php", "ext":"php"}'>php</option>
+                        <option value='{"name":"python", "ext":"py"}'>python</option>
+                        <option value='{"name":"r", "ext":"r"}'>r</option>
+                        <option value='{"name":"ruby", "ext":"rb"}'>ruby</option>
+                        <option value='{"name":"sh", "ext":"sh"}'>sh</option>
+                        <option value='{"name":"sql", "ext":"sql"}'>sql</option>
+                    </select>
+                </div>
+            </div>
+            <div class="form-group" id="snippet-editor">
+                <label class="form-label-header">Code</label>
+                <div class="form-group_item">
+                    <div id="editor_div"></div>
+                    <textarea name='code'>
+function example_code_snippet(items) {
+    var x = "This is javascript. Select a file-type for different syntax highlighting";
+    return x;
+}</textarea>
+                </div>
+            </div>
+            
+            
+            
+            
+            
+            
+          </div>
+      </div>
+      <div class="modal-footer">
+        <div id="ace_output"></div>
+        <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+        <button id="btn_post_related_gist" class="btn btn-primary" type="button" data-module="post_related_gist"
+                  data-module-dataset_id="{{ res.package_id }}"
+                  data-module-resource_id="{{ res.id }}"
+                  data-module-resource_name="{{ res.name }}"
+                  data-module-html="post_related_gist.html"
+                  data-module-title="{{ _('posting code') }}">
+            Submit code snippet
+            <span class="cf-icon cf-icon-update cf-icon__pulse hidden" id="gist-loading"></span>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
       
   {% endif %}
 {% block resource_view_nav %}
 {% endblock %}
 {% block resource_view_content %}
-<section class="module">
-<div class="module-content">
-    <hr>
-<h1>{{_('Dataset status')}}</h1>
-{{_('Information about the operational health of a resource ("operational metadata") is stored for analysis and visualization in the following table.')}}
```

Comment:
TBH I got rid of them because there wasn't a good place to put them. With the expandables there's a title in the clickable part and then content in the rest.

But now that I look at it I'm not convinced we need them. "Dataset status" admittedly isn't very descriptive but once they place with the tool they should learn what the sections are... I think?